pkg/utils/ocmWrapper.go

```diff
@@ -261,6 +262,18 @@ func (*DefaultOCMInterfaceImpl) IsProduction() (bool, error) {
 	return connection.URL() == "https://api.openshift.com", nil
 }
 
+func (*DefaultOCMInterfaceImpl) GetStsSupportJumpRole(clusterID string) (string, error) {
```

Comment:
Minor Nitpick: Consider renaming this function to `GetStsSupportJumpRoleARN`.

Reply:
My original thought was to go with the name of the OCM call itself, which is "StsSupportJumpRole", but I think specifying that it's the "ARN" is a good call here. I'll rename it.